db.py

```diff
@@ -1,21 +1,32 @@
 import sqlite3
+from peewee import SqliteDatabase
 import os
 
+# Initialize Peewee database
+db = SqliteDatabase('remo.db')
+
 class Database:
     def __init__(self):
-        self.conn = sqlite3.connect('remo.db')
-        self.cur = self.conn.cursor()
+        self.db = db  # Use Peewee's database instance
         self._init_db()
 
     def _init_db(self):
+        # Connect to the database
+        self.db.connect()
         files = os.listdir('sql')
         for file in files:
             with open(f'sql/{file}', 'r') as f:
                 query = f.read()
-                self.cur.execute(query)
+                # Use Peewee's execution method
+                self.db.execute_sql(query)
 
     def query(self, query):
-        self.cur.execute(query)
```

Comment:
seems like these methods are unnecessary now that we are using entity to insert and read data?

Reply:
This part here creates the tables. I didn't test it without it but I definitely deleted the Db instance file and had it regenerate when starting the app.